sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -877,13 +879,18 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product with Tre
       t.copy(properties = Utils.redact(t.properties).toMap,
         options = Utils.redact(t.options).toMap) :: Nil
     case table: CatalogTable =>
-      table.storage.serde match {
-        case Some(serde) => table.identifier :: serde :: Nil
-        case _ => table.identifier :: Nil
-      }
+      stringArgsForCatalogTable(table)
+
     case other => other :: Nil
   }.mkString(", ")
 
+  private def stringArgsForCatalogTable(table: CatalogTable): Seq[Any] = {
+    table.storage.serde match {
+      case Some(serde) => table.identifier :: serde :: Nil
```

Comment:
I see inconsistence on representing table identifier on the plan - while this produces "quoted" string, I also see existing codebase to produce "unquoted" string. e.g. LogicalRelation.

I'd like to hear the voice which is our preference. For now I used "unquoted" string for other places and leave this as it is (hence "quoted" string).

Reply:
I think quoted string is better, let's keep it as it was first.

Reply:
OK. Let me leave this as it is, and see whether I can make it consistent for other places in the following work (out of this PR).